plugins/azure/virtualmachinescaleset/vmssApplicationGatewayEnabled.js

```diff
@@ -0,0 +1,70 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM Scale Set Application Gateway Enabled',
+    category: 'Virtual Machine Scale Set',
+    domain: 'Compute',
+    description: 'Ensures that Azure Virtual Machine scale sets has Application Gateway enabled.',
```

Comment:
i think application gateway is not something that can be enabled. instead its something that we configure